package.json

```diff
@@ -10,23 +10,23 @@
     "benchmark": "npx concurrently -k -s first \"node ./examples/benchmark/simple.js\" \"npx autocannon -c 100 -d 30 -p 10 localhost:3000/\"",
     "benchmark:parser": "npx concurrently -k -s first \"node ./examples/benchmark/parser.js\" \"npx autocannon -c 100 -d 30 -p 10 -i ./examples/benchmark/body.json -H \"content-type:application/jsoff\" -m POST localhost:3000/\"",
     "build:validation": "node build/build-error-serializer.js && node build/build-validation.js",
-    "coverage": "cross-env NODE_OPTIONS=no-network-family-autoselection npm run unit -- --cov --coverage-report=html",
-    "coverage:ci": "npm run unit -- --cov --coverage-report=html --no-browser --no-check-coverage",
-    "coverage:ci-check-coverage": "nyc check-coverage --branches 100 --functions 100 --lines 100 --statements 100",
+    "coverage": "npm run unit -- --coverage-report=html",
+    "coverage:ci": "npm run unit -- --coverage-report=html --no-browser --no-check-coverage",
+    "coverage:ci-check-coverage": "c8 check-coverage --branches 100 --functions 100 --lines 100 --statements 100",
```

Comment:
```suggestion
    "coverage:ci-check-coverage": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100",
```

Reply:
Are you sure? According to the c8 docs, this is the right command.

https://github.com/bcoe/c8#checking-coverage

Reply:
Did not see that it has a command-like stuff - if it can be an argument, it seems more easy to find and understand 😅